Documentation~/Installation.md

```diff
@@ -1,6 +1,18 @@
 # Installation
 
-## Automatic Installation
+## Automatic Installation via DCC tools
```

Comment:
Perhaps "Unity Project Setup" via DCC tools is a better naming here ?

Reply:
done